cmake/Platform/Linux/runtime_dependencies_linux.cmake.in

```diff
@@ -9,23 +9,32 @@
 cmake_policy(SET CMP0012 NEW) # new policy for the if that evaluates a boolean out of "if(NOT ${same_location})"
 
 function(ly_copy source_file target_directory)
-    get_filename_component(target_filename "${source_file}" NAME)
-    get_filename_component(target_filename_ext "${source_file}" LAST_EXT)
-    cmake_path(COMPARE "${source_file}" EQUAL "${target_directory}/${target_filename}" same_location)
+    cmake_path(GET source_file FILENAME target_filename)
+    cmake_path(APPEND target_file "${target_directory}" "${target_filename}")
+    cmake_path(COMPARE "${source_file}" EQUAL "${target_file}" same_location)
     if(NOT ${same_location})
-        file(LOCK ${target_directory}/${target_filename}.lock GUARD FUNCTION TIMEOUT 300)
-        if("${source_file}" IS_NEWER_THAN "${target_directory}/${target_filename}")
+        file(LOCK ${target_file}.lock GUARD FUNCTION TIMEOUT 300)
+        file(SIZE "${source_file}" source_file_size)
+        if(EXISTS "${target_file}")
+            file(SIZE "${target_file}" target_file_size)
+        else()
+            set(target_file_size 0)
+        endif()
+        if((NOT source_file_size EQUAL target_file_size) OR "${source_file}" IS_NEWER_THAN "${target_directory}/${target_filename}")
+            message(STATUS "Copying \"${source_file}\" to \"${target_directory}\"...")
             file(COPY "${source_file}" DESTINATION "${target_directory}" FILE_PERMISSIONS @LY_COPY_PERMISSIONS@ FOLLOW_SYMLINK_CHAIN)
-            
+            file(TOUCH_NOCREATE ${target_directory}/${target_filename})
```

Comment:
Same here.

Same with the `file(RPATH_CHANGE FILE "${target_directory}/${target_filename}" ...` below.